src/lib.rs

```diff
@@ -190,14 +189,15 @@ fn run_fixing_loop(
         let mut instantiated_per_file_rules: HashMap<RuleName, Box<dyn RuleInstancePerFile>> =
             Default::default();
         tree_sitter_grep::run_for_slice_with_callback(
-            file_contents,
+            &**file_contents,
+            None,
```

Comment:
So this (inside the "fixing loop") is actually the place where I was thinking it would probably be more optimal to actually store the fixed file contents as a `Rope` and keep track of/pass an incrementally pre-parsed `tree_sitter::Tree` (into `tree_sitter_grep`)

(and this entry point is already ready to take those arguments)

Reply:
Thought about this more and I think it's "overkill" to convert into a `Rope` in the "normal" (ie from the command-line entry point) "fixing loop"

Because it's probably the majority of the time that you only go through the fixing loop once, and so it seems like the cost of converting to/from a `Rope` would outweigh the extra cost of applying the mutations to a `String` instead of a `Rope`?